stanford-earth/SE3_BLT
# EARTH-515: Update research areas to remove default page title.

# READY FOR REVIEW

# Summary
- Removes page title from rendering as a block and a region on research_area types

# Needed By (Date)
- End of sprint

# Steps to Test

1. Check out branch EARTH-515 of SE3_BLT, stanford_research_area, matson
2. Pull in the config settings `drush cim -y` 
3. Revert the feature `drush features-import stanford_research_area -y`
4. Clear cache `drush cr` and visit `/securing-energy-future`
5. Ensure the hero image is at the top of the page and we are not getting double page titles.

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
